frontend/src/metabase/dashboard/components/DashboardHeader.tsx

```diff
@@ -90,7 +90,11 @@ const DashboardHeader = ({
         className="Header-buttonSection"
         isNavBarOpen={isNavBarOpen}
       >
-        {headerButtons}
+        {headerButtons.map((headerButton, index) => (
+          <div style={{ display: "contents" }} key={index}>
```

Comment:
why not to use import from "styled"?

Reply:
1) you shouldn't ever use indexes for keys in react because it can cause some insane and really hard to track down bugs if the order of the items ever changes
2) if some of these items are missing keys, I think it would be better to add them wherever `headerbuttons` is coming from

Reply:
Ah yeah, it's a horrible practice - I basically did this just to make sure I wouldn't see that error. I was trying to add the keys to the header buttons, but I was seeing a few `Fragment` components so I didn't want to spend time adding the keys when the code was already working fine before, especially when changing that code was also not really in the scope of this PR

Reply:
From what I see, the options seem to be:
- leave it and add the keys and things later in a different PR
- Make the changes now
I just don't want to muddle the PR - let me know what you tink

Reply:
Ok - seems like it was just adding one key to the `Fragment` for the `Add actions` button, so I just added it and seems to be fine